backend/src/services/user/user.service.ts

```diff
@@ -85,28 +85,15 @@ class User {
       id: user.id,
       email: user.email,
       createdAt: user.createdAt,
-      fullName,
+      userDetails,
       permissions: [],
     };
   }
 
   public async getByEmail(
     email: string,
   ): Promise<UsersByEmailResponseDto | null> {
-    const user = await this.#userRepository.getByEmail(email);
-
-    if (!user) {
-      return null;
-    }
-
-    return {
-      id: user.id,
-      email: user.email,
-      fullName: user.fullName,
-      passwordHash: user.passwordHash,
-      passwordSalt: user.passwordSalt,
-      createdAt: user.createdAt,
-    };
+    return await this.#userRepository.getByEmail(email);
```

Comment:
```suggestion
    return this.#userRepository.getByEmail(email);
```
Also, remove `async` from the method declaration

Reply:
thanks @fshabanov